lib/ansible/modules/crypto/openssl_privatekey.py

```diff
@@ -23,6 +23,11 @@
       L(ECC,https://en.wikipedia.org/wiki/Elliptic-curve_cryptography)
       private keys.
     - Keys are generated in PEM format.
+    - "Please note that the module regenerates private keys if they don't match
```

Comment:
```suggestion
    - "Take note that the module regenerates private keys if they don't match
```

Reply:
What's wrong with "Please note"?

Reply:
While I appreciate the courtesy, it raises the question if/where we should write "please" elsewhere and how often.